test_cases/test_dojo.py

```diff
@@ -239,6 +243,47 @@ def test_load_people_invalid_entry(self):
 		result = self.the_dojo.load_people("LoadFile")
 		self.assertEqual(result, "Invalid entry!")	
 
+	def test_save_state_database_file_exists(self):
+		self.the_dojo.save_state("sampleDB")
+		self.assertTrue(os.path.isfile("sampleDB.db"))
+		#os.remove("sampleDB.db")
+
+	def test_save_state_successfully_saves_room_data_to_DB(self):
+		self.the_dojo.create_room("office", "Red")
+		self.the_dojo.save_state("sampleDB")
+		#check if data saved exists in db
+		engine = create_engine('sqlite:///sampleDB.db')
+		Base.metadata.bind = engine
+		DBSession = sessionmaker(bind=engine)
+		session = DBSession()
+		existing = session.query(RoomModel).filter_by(room_name="Red").first()
+		self.assertTrue(existing)
+
+	def test_save_state_successfuly_saves_person_data_to_DB(self):
+		self.the_dojo.add_person("Pete", "pete@pete", "fellow", "y")
+		self.the_dojo.save_state("sampleDB")
+		engine = create_engine('sqlite:///sampleDB.db')
+		Base.metadata.bind = engine
```

Comment:
According to this documentation:
http://docs.sqlalchemy.org/en/latest/orm/tutorial.html
metadata is used for the creation of tables. Therefore, I do not think it necessary in your tests.